clean-abap/CleanABAP.md

```diff
@@ -3607,6 +3608,24 @@ not enforce writing ABAP Doc for each and everything.
 > Read more in _Chapter 4: Good Comments: Javadocs in Public APIs_ and _Chapter 4: Bad Comments:
 > Javadocs in Nonpublic Code_ of [Robert C. Martin's _Clean Code_].
 
+### Prefer pragmas to pseudo comments
+
+> [Clean ABAP](#clean-abap) > [Content](#content) > [Comments](#comments) > [This section](#prefer-pragmas-to-pseudo-comments)
+
+Prefer pragmas to pseudo comments to blend out irrelevant warnings and errors identified by the ATC. Pseudo comments 
```

Comment:
Better _suppress_ instead of _blend out_? While _blend out_ in theory works, it's not a common translation for this figure of speech, see for example the forum discussions under https://dict.leo.org/german-english/ausblenden.

Reply:
Thanks, I've changed the wording as suggested.